examples/xgboost/train.py

```diff
@@ -15,6 +15,8 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description='XGBoost example')
+    parser.add_argument('--learning_rate', type=float, default=0.3,
```

Comment:
could you change this to `--learning-rate` to match the convention here? (It shouldn't change line 48's `args.learning_rate,` though.) thanks!

Reply:
done @sueann